src/libecalc/presentation/yaml/file_context.py

```diff
@@ -0,0 +1,50 @@
+from dataclasses import dataclass
+from typing import Optional
+
+from libecalc.presentation.yaml.yaml_entities import YamlDict
+
+
+@dataclass
+class FileMark:
+    line_number: int
+    column_number: int
+
+    def __str__(self) -> str:
+        return f" line {self.line_number}, column {self.column_number}"
+
+
+@dataclass
+class FileContext:
+    start: FileMark
+    end: Optional[FileMark] = None
+    name: Optional[str] = None
+
+    def __str__(self) -> str:
+        file_context_string = ""
+        if self.name is not None:
+            file_context_string += f" in '{self.name}',"
+
+        file_context_string += str(self.start)
```

Comment:
not include end, if set?

Reply:
Not sure, seems a bit confusing to me. We might have to be more specific in the errors here, as sometimes start may mean "the error is here" and other times it means the error is between start and end